examples/tv-casting-app/tv-casting-common/src/TargetVideoPlayerInfo.cpp

```diff
@@ -60,11 +71,14 @@ CHIP_ERROR TargetVideoPlayerInfo::Initialize(NodeId nodeId, FabricIndex fabricIn
 CHIP_ERROR TargetVideoPlayerInfo::FindOrEstablishCASESession(std::function<void(TargetVideoPlayerInfo *)> onConnectionSuccess,
                                                              std::function<void(CHIP_ERROR)> onConnectionFailure)
 {
-    mOnConnectionSuccessClientCallback = onConnectionSuccess;
-    mOnConnectionFailureClientCallback = onConnectionFailure;
-    Server * server                    = &(chip::Server::GetInstance());
-    server->GetCASESessionManager()->FindOrEstablishSession(ScopedNodeId(mNodeId, mFabricIndex), &mOnConnectedCallback,
-                                                            &mOnConnectionFailureCallback);
+    ChipLogProgress(AppServer, "TargetVideoPlayerInfo::FindOrEstablishCASESession called");
+
+    VideoPlayerConnectionContext * connectionContext = new VideoPlayerConnectionContext(
```

Comment:
Where is this VideoPlayerConnectionContext destroyed?

Reply:
See line #136 and #161 in examples/tv-casting-app/tv-casting-common/include/TargetVideoPlayerInfo.h